library/src/scripts/sectioning/Tabs.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement, useState } from "react";
+import { Tabs, TabList, Tab, TabPanels, TabPanel } from "@reach/tabs";
+import { tabClasses } from "@library/sectioning/TabStyles";
+
+interface IData {
+    label: string;
+    panelData: string;
+    contents: React.ReactNode;
+}
+interface IProps {
+    data: IData[];
+}
+
+export function DataTabs(props: IProps) {
```

Comment:
It's a bit odd to call this component `DataTabs` when the name of the file is `Tabs`. We should call it `Tabs` and adjust the import above.

Eg.:

```ts
import { Tabs as ReachTabs } from "@reach/tabs";
```